app/views/validators/index.html.erb

```diff
@@ -304,13 +304,25 @@
                       else
                         'Gray'
                       end
-                    slot_vector = validator.score&.skipped_slot_history.to_a[-vl..-1].map {|v| (v.to_f*100.0).round(1)}
-                 %>
+                    slot_vector = validator.score&.skipped_slot_history.to_a[-vl..-1].map { |v| (v.to_f*100.0).round(1) }
+
+                    moving_avg_color = if validator.skipped_slot_score == 2
+                        '#7fbf7f'
+                      elsif validator.skipped_slot_score == 1
+                        '#7f7fff'
+                      else
+                        '#bfbfbf'
+                      end
```

Comment:
These colors are just darker shades of the existing colors. @kbiala feel free to make any changes you recommend whenever you get to this.